### ISSUE TITLE
Sequelize-auto not generating models for mysql db views

### ISSUE THREAD

**Issue by FrancaTM** [sequelize/sequelize-auto#579]
When I run:
`sequelize-auto -o "./app/models" -d sample_db -h localhost -u root -x root -e mysql -v`

Sequelize-auto seems to work fine, but it's not generating models for 2 views in my db and during the process it throws this error for both views:
`Error: No description found for "[object Object]" table. Check the table name and schema; remember, they _are_ case sensitive.`
`    at MySQLQueryInterface.describeTable (/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/abstract/query-interface.js:378:15)`
`    at processTicksAndRejections (node:internal/process/task_queues:96:5)`

I'm not sure if the cause of the problem is something within sequelize-auto or sequelize and I did try to check the names and schema of db. Maybe I'm missing something, I don't know.

I just started using sequelize-auto and I'm sorry if this is not the right place to post this kind of issue, but I already tried to search the errors and other possible reasons for this, but with no success. I would really appreciate any help from someone who faced this problem before. :/

Below is the the output log (with some cleaning):
`[...] sample-backend % sequelize-auto -o "./app/models" -d sample_db -h localhost -u root -x root -e mysql -v`
[...]
`{`
`  spaces: true,`
`  indentation: 2,`
`  directory: './app/models',`
`  additional: {},`
`  dialect: 'mysql',`
`  port: 3306,`
`  host: 'localhost',`
`  database: 'sample_db',`
`  storage: 'sample_db',`
`  tables: null,`
`  skipTables: null,`
`  skipFields: null,`
`  pkSuffixes: null,`
`  schema: undefined,`
`  lang: 'es5',`
`  caseModel: 'o',`
`  caseFile: 'o',`
`  caseProp: 'o',`
`  noAlias: false,`
`  noInitModels: false,`
`  noWrite: false,`
`  views: true,`
`  singularize: false,`
`  username: 'root',`
`  useDefine: false`
`}`
`Executing (default): SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_SCHEMA = 'sample_db';`
`Executing (default): select table_name from information_schema.tables where table_type = 'VIEW' and table_schema = 'sample_db'`
[...]
`Executing (default): SELECT K.CONSTRAINT_NAME as constraint_name`
`      , K.CONSTRAINT_SCHEMA as source_schema`
`      , K.TABLE_NAME as source_table`
`      , K.COLUMN_NAME as source_column`
`      , K.REFERENCED_TABLE_SCHEMA AS target_schema`
`      , K.REFERENCED_TABLE_NAME AS target_table`
`     , K.REFERENCED_COLUMN_NAME AS target_column`
`      , C.EXTRA AS extra`
`      , C.COLUMN_KEY AS column_key`
`      FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS K`
`     LEFT JOIN INFORMATION_SCHEMA.COLUMNS AS C`
`        ON C.TABLE_NAME = K.TABLE_NAME AND C.COLUMN_NAME = K.COLUMN_NAME AND C.TABLE_SCHEMA = K.CONSTRAINT_SCHEMA`
`      WHERE K.TABLE_NAME = '[object Object]'`
`             AND C.TABLE_SCHEMA = 'sample_db'`
[...]
`Executing (default): SHOW FULL COLUMNS FROM 'Employee';`
`Executing (default): SHOW FULL COLUMNS FROM '[object Object]';`
[...]
`Executing (default): SHOW INDEX FROM 'Employee'`
`Error: No description found for "[object Object]" table. Check the table name and schema; remember, they _are_ case sensitive.`
`    at MySQLQueryInterface.describeTable (/usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/abstract/query-interface.js:378:15)`
`    at processTicksAndRejections (node:internal/process/task_queues:96:5)`
[...]

**Comment by steveschmitt** [comment]
That is strange!

What output do you get if you manually run this sql query?:
```sql
select table_name from information_schema.tables where table_type = 'VIEW' and table_schema = 'sample_db'
```

**Comment by FrancaTM** [comment]
That query returns a column (TABLE_NAME) with the names of my 2 table views, all lowercase. 

![Screen Shot 2021-12-03 at 19 36 14](https://user-images.githubusercontent.com/14852938/144681887-65833712-080a-4c23-b9e6-04394703ef35.png)

First I thought the problem could be something related with case sensitivity or something like that, but I did some tests, including trying to generate models from lowercase table names and uppercase view names, the first went fine, the second didn't. Could be something done wrong during the creation/definition of my view query? Just some random guessing.

**Comment by steveschmitt** [comment]
There might be a bug in the way that sequelize-auto passes the view names to the next query, but I can't reproduce the problem.

What versions of **sequelize** and **sequelize-auto** are you using?

**Comment by FrancaTM** [comment]
Hey @steveschmitt, I think I found the culprit for the issue.

The log for your request (with some cleaning) was this:
`[`
`  [...]`
`  'Employee',`
`  { TABLE_NAME: 'publicsearch' },`
`  { TABLE_NAME: 'unityinformation' }`
`]`
`[`
`  [...]`
`  { table_name: 'Employee', table_schema: null },`
`  { table_name: '[object Object]', table_schema: null },`
`  { table_name: '[object Object]', table_schema: null }`
`]`

I also tested the changes in the commit you've made (`TABLE_NAME as table_name`) and that worked fine for me. Besides that, I found a way to test sequelize-auto in another machine and the package worked fine without any changes. 

The main difference was the way mysql was installed. I did the installation through brew, but the other one was made through MAMP. I'm not sure, but I think the problem could be some case-sensitivity configuration of my mysql installation.

The outputs of the `init-models.js` table relationships were different. On my machine, the output of the names was a mix of uppercase and lowercase letters, on the other hand, the other machine showed just lowercase letters. Anyway, my bad I didn't check that before. Thank you very much @steveschmitt for your help.

**Comment by steveschmitt** [comment]
Great, I'm glad it worked.  

I'm wondering now if the mix of uppercase and lowercase letters in `init-models` will cause a problem for some users.

**Comment by FrancaTM** [comment]
Just some additional info...

I completely removed mysql from my system and just installed MAMP instead. Now this is the new log as per your request:

`[...]`
`Executing (default): SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_SCHEMA = 'retorno_rapido_db';`
`Executing (default): select TABLE_NAME as table_name from information_schema.tables where table_type = 'VIEW' and table_schema = 'retorno_rapido_db'`
`[`
`  [...]`
`  'Employee',`
`  { table_name: 'publicsearch' },`
`  { table_name: 'unityinformation' }`
`]`
`[`
`  [...]`
`  { table_name: 'Employee', table_schema: null },`
`  { table_name: 'publicsearch', table_schema: null },`
`  { table_name: 'unityinformation', table_schema: null }`
`]`

Now everything is ok here, and to double check I also tested the changes of your commit and both codes (with and without the changes) seemed to work fine.

Regarding the mix of uppercase and lowercase letters in `init-models.js`, I can't remember if I checked for some possible problems with my db before removing mysql and installing MAMP, I'm sorry :/

**Comment by steveschmitt** [comment]
Fixed in sequelize-auto@0.8.7
